backend/src/handlers/reviews/main.go

```diff
@@ -85,50 +89,41 @@ func getUserReview(ctx context.Context, req Request) (Response, error) {
 	}
 
 	return Response{StatusCode: 200, Body: body, Headers: views.DefaultHeaders}, nil
-
-	// conrmad is distracting me
-	// i came here to cause problems
-	// me me mow me mow
-	// can we leave these in here
```

Comment:
you heard the PM.

Reply:
lmao i got annoyed of scrolling up and down on a trackpad

Reply:
yeah ive been there :sob: they can go lol

Reply:
![image](https://user-images.githubusercontent.com/42351353/222968021-0771a472-d36e-4623-9689-66f608969b63.png)